src/NUnitFramework/framework/Assume.cs

```diff
@@ -16,7 +16,7 @@ namespace NUnit.Framework
     /// result. If an assumption is not met, the test
     /// should produce an inconclusive result.
     /// </summary>
-    public class Assume
+    public static class Assume
```

Comment:
We shouldn't make public classes outside the Internal namespace static. A publicized technique for "extending" the NUnit assertion classes is to do this:

```cs
public class Assert : NUnit.Framework.Assert
{
    public static void Foo(...) { }
}
```

Reply:
We could consider making them `abstract` to break code which does `new Assert()` and I thought we had done so.

Reply:
so revert the Assume and Warn changes?